rudder-core/src/main/scala/com/normation/rudder/domain/policies/RuleTarget.scala

```diff
@@ -232,12 +232,11 @@ object RuleTarget extends Loggable {
       case AllTargetExceptPolicyServers => nodes ++ allNodes.collect { case(k,n) if(!n._1) => k }
       case PolicyServerTarget(nodeId) => nodes + nodeId
       case AllServersWithRole =>
-        //we have a special case for the root node that always belongs to that group, even if some weird
-        //scenario lead to the removal (or non addition) of them
-        nodes ++ allNodes.collect { case(k,n) if(n._2.size>0 || n._1 == Constants.ROOT_POLICY_SERVER_ID) => k }
+        // all nodes with server roles or policyServer (ie relay)
+        nodes ++ allNodes.collect { case(k,n) if(n._2.size>0 || n._1) => k }
```

Comment:
ho, so it was before in another file ( see PR https://github.com/Normation/rudder/pull/824/files ) and that broke it.